content/en/docs/virtualization.md

```diff
@@ -397,3 +397,170 @@ Creating a virtual router requires a Cozystack administrator account.
+
+Follow the below steps.
+
+Label the worker node explicitly for GPU passthrough workloads:
+
+```bash
+kubectl label node <node-name> --overwrite nvidia.com/gpu.workload.config=vm-passthrough
+```
+
+Install the GPU Operator, by enabling the GPU operator bundle in your Cozystack configuration.
+
+```bash
+kubectl edit -n cozy-system configmap cozystack-config
+```
+
+Add `gpu-operator` to the list of bundle-enabled packages:
+
+```yaml
+bundle-enable: gpu-operator
+```
+
+The following operands get deployed. Ensure all pods are in a running state and all validations succeed with the sandbox-validator component:
+
+```bash
+kubectl get pods -n cozy-gpu-operator
+```
+
+Example output:
+
+```console
+NAME                                            READY   STATUS    RESTARTS   AGE
+...
+nvidia-sandbox-device-plugin-daemonset-4mxsc    1/1     Running   0          40s
+nvidia-sandbox-validator-vxj7t                  1/1     Running   0          40s
+nvidia-vfio-manager-thfwf                       1/1     Running   0          78s
+```
+
+You can use `kubectl debug node` or `kubectl node-shell -x` to access the node and check the GPU status.
+
+```bash
+lspci --nnk -d 10de:
+```
+
+The vfio-manager pod will bind all GPUs on the node to the vfio-pci driver. Example output:
+
+```console
+3b:00.0 3D controller [0302]: NVIDIA Corporation Device [10de:2236] (rev a1)
+       Subsystem: NVIDIA Corporation Device [10de:1482]
+       Kernel driver in use: vfio-pci
+86:00.0 3D controller [0302]: NVIDIA Corporation Device [10de:2236] (rev a1)
+       Subsystem: NVIDIA Corporation Device [10de:1482]
+       Kernel driver in use: vfio-pci
+```
+
+The sandbox-device-plugin will discover and advertise these resources to kubelet. In this example, we have two A10 GPUs:
+
+```bash
+# kubectl describe node <node-name>
+```
+
+Example output:
+
+```console
+...
+Capacity:
+  ...
+  nvidia.com/GA102GL_A10:         2
+  ...
+Allocatable:
+  ...
+  nvidia.com/GA102GL_A10:         2
+...
+```
+
+{{% alert color="info" %}}
+The resource name is currently constructed by joining the device and device_name columns from the [PCI IDs database](https://pci-ids.ucw.cz/v2.2/pci.ids).
+For example, the entry for A10 in the database reads `2236  GA102GL [A10]`, which results in a resource name `nvidia.com/GA102GL_A10`.
+{{% /alert %}}
+
+### Update the KubeVirt CR
+
+Next, we will update the KubeVirt Custom Resource, as documented in the [KubeVirt user guide](https://kubevirt.io/user-guide/virtual_machines/host-devices/#listing-permitted-devices),
+so that the passthrough GPUs are permitted and can be requested by a KubeVirt VM.
+Note, replace the values for `pciVendorSelector` and `resourceName` to correspond to your GPU model.
+We set `externalResourceProvider=true` to indicate that this resource is being provided by an external device plugin,
+in this case the `sandbox-device-plugin` which is deployed by the Operator.
+
+```yaml
+kubectl edit kubevirt -n kubevirt
```

Comment:
This will work for now, but these changes may be reverted by Flux in the future.